src/jsonGraphToCypher.js

```diff
@@ -106,14 +103,14 @@ const jsonGraphToCypher = async (graph, sourceGraphId) => {
       if (isLocation) {
         rps.push(`${nodeName}.point = point({latitude:toFloat(${props.latitude}), longitude:toFloat(${props.longitude})})`);
       }
-      typedProperties = `${rps.join(', ')}, `;
+      typedProperties = `,  ${rps.join(', ')}`.trim();
     }
-    const nodeLabel = nodeToNodeLabel(node);
-    const typeNode = isTypeNode(node, graph.links);
+    const [typeNode, nodeLink] = isTypeNode(node, graph.links);
+    const nodeLabel = nodeToNodeLabel(node, nodeLink);
     if (typeNode) {
       query += `MERGE ( ${nodeName} : \`Type\` { id: "${node.id}" }) SET ${nodeName}.type = "${node.id.split('/').pop().split('#').pop()}", ${typedProperties} ${nodeName}.sourceTimestamp = datetime() ${sourceGraphInfo.replace(', ', `, ${nodeName}.`).replace(':', ` =`)}\n`;
     } else {
-      query += `MERGE ( ${nodeName} : \`${nodeLabel.toString()}\` { id: "${node.id}" }) SET ${typedProperties}${nodeName}.sourceTimestamp = datetime() ${sourceGraphInfo.replace(', ', `, ${nodeName}.`).replace(':', ` =`)}\n`;
+      query += `MERGE ( ${nodeName} : \`${nodeLabel.toString()}\` { id: "${node.id}" }) SET ${typedProperties && `${typedProperties.substring(2)},`} ${nodeName}.sourceTimestamp = datetime() ${sourceGraphInfo.replace(', ', `, ${nodeName}.`).replace(':', ` =`)}\n`;
```

Comment:
ah ok I see we are removing comma here if string is present, seems like we could simplify to not have comma in first place and always have at the beginning though?